### PR TITLE
ticdc: update java example link

### PR DESCRIPTION
**Pull request by hi-rustin**
Signed-off-by: hi-rustin <rustin.liu@gmail.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

update java example link

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): cc: https://github.com/pingcap/ticdc/pull/3547
 
### Do your changes match any of the following descriptions?

None

### CODE REVIEW

**Comment by hi-rustin** [comment]
/cc @amyangfei @asddongmen 

Could you please take a look? Thanks!

**Review by amyangfei — APPROVED** [review]

File: ticdc/ticdc-open-protocol.md [hunk 1]
```diff
@@ -274,10 +274,10 @@ COMMIT;
 
 ## 消费端协议解析
 
-目前 TiCDC 没有提供 Open Protocol 协议解析的标准实现，但是提供了 Golang 版本和 Java 版本的解析 demo。用户可以参考本文档提供的数据格式和以下 demo 实现消费端协议解析。
+目前 TiCDC 没有提供 Open Protocol 协议解析的标准实现，但是提供了 Golang 版本和 Java 版本的解析例子。用户可以参考本文档提供的数据格式和以下例子实现消费端协议解析。
```
Comment (TomShawn) [inline ticdc/ticdc-open-protocol.md:277]:
```suggestion
目前 TiCDC 没有提供 Open Protocol 协议解析的标准实现，但是提供了 Golang 版本和 Java 版本的解析例子。你可以参考本文档提供的数据格式和以下例子实现消费端协议解析。
```

Comment (asddongmen) [reply ticdc/ticdc-open-protocol.md:277]: 把 ”你“ 也去掉会不会更好一点？

**Review by asddongmen — APPROVED** [review]

**Review by Liuxiaozhen12 — APPROVED** [review]

**Pull request merged**